src/data/businesses.json

```diff
@@ -28,33 +28,38 @@
   "medFindIt": {
     "id": "medFindIt",
     "name": "Med Find It",
-    "ownerId": null,
+    "ownerId": "Greg Malonas",
```

Comment:
ownerIds need to match ids in https://github.com/markalanrichards/ownemployed/blob/master/src/data/users.json

Reply:
@lyonjinx to add a bunch of users to the database quickly, the minimum data they need is an ID and a name, and the relevant businessIds that they are connected to. It's ok if the rest is missing, e.g.

```
{
  "gregMalonas": {
    "id": "gregMalonas",
    "name: "Greg Malonas",
    "ownedModels": ["medFindIt"]
  }
}
```

Reply:
done. sent for review